snow/engine/common/test_engine.go

```diff
@@ -354,3 +358,16 @@ func (e *EngineTest) Chits(validatorID ids.ShortID, requestID uint32, containerI
 	}
 	return nil
 }
+
+// IsBootstrapped ...
+func (e *EngineTest) IsBootstrapped() bool {
+	if e.IsBootstrappedF != nil {
+		return e.IsBootstrappedF()
+	} else if e.CantIsBootstrapped {
+		if e.T != nil {
+			e.T.Fatalf("Unexpectedly called IsBootstrapped")
+		}
+		panic("Unexpectedly called IsBootstrapped")
+	}
+	return e.IsBootstrappedF()
```

Comment:
Should return a reasonable default value here. Probably false, as that's typically the zero value for bools.